src/Sylius/Bundle/ResourceBundle/Controller/RequestConfigurationFactory.php

```diff
@@ -86,4 +87,13 @@ private function parseApiParameters(Request $request)
 
         return array_merge($request->attributes->get('_sylius', []), $parameters);
     }
+
+    /**
+     * @return array
+     */
+    public function getDefaultParameters()
```

Comment:
Why do we need this public method? Let's use the property. If we want to decouple this responsibility, then let's think about DefaultSettingsProviderInterface in next iteration or something. :) For now I'd just use the class property directly when ``array_merge`` is called.